src/main/java/org/terasology/multiBlock/Basic3DSizeFilter.java

```diff
@@ -16,9 +16,9 @@
 package org.terasology.multiBlock;
 
 import com.google.common.base.Predicate;
-import org.terasology.math.geom.Vector3i;
+import org.joml.Vector3i;
 
-public class Basic3DSizeFilter implements Predicate<org.joml.Vector3i> {
+public class Basic3DSizeFilter implements Predicate<Vector3i> {
```

Comment:
I think this should be
```suggestion
public class Basic3DSizeFilter implements Predicate<Vector3ic> {
```